web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/DownloadImageService.java

```diff
@@ -0,0 +1,67 @@
+
+package edu.unc.lib.boxc.web.services.processing;
+
+import edu.unc.lib.boxc.model.api.DatastreamType;
+import edu.unc.lib.boxc.search.api.models.ContentObjectRecord;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Objects;
+
+/**
+ * @author snluong
+ */
+public class DownloadImageService {
+    private String iiifBasePath;
+
+    public ResponseEntity<InputStreamResource> streamImage(String pidString, String size)
```

Comment:
Generally we want to have javadoc for public methods, like this and getSize. We usually skip it for real getters and setters though.